examples/pytorch/nlp/huggingface_models/question-answering/quantization/ptq_static/ipex/README.md

```diff
@@ -32,6 +34,69 @@ python run_qa.py
 >
 > /path/to/checkpoint/dir is the path to finetune output_dir
 
+## 2. Quantization with XPU
+Please build an IPEX docker container with following steps. Please also refer to the [official guide](https://github.com/intel/intel-extension-for-pytorch/tree/xpu-master/docker).
+#### 2.1 Build Container and Environment Variables
+```bash
+wget https://raw.githubusercontent.com/intel/intel-extension-for-pytorch/xpu-master/docker/Dockerfile.xpu
+wget https://raw.githubusercontent.com/intel/intel-extension-for-pytorch/xpu-master/docker/build.sh
+./build.sh xpu-flex
+
+export IMAGE_NAME=intel/intel-extension-for-pytorch:xpu-flex
+export VIDEO=$(getent group video | sed -E 's,^video:[^:]*:([^:]*):.*$,\1,')
+export RENDER=$(getent group render | sed -E 's,^render:[^:]*:([^:]*):.*$,\1,')
+test -z "$RENDER" || RENDER_GROUP="--group-add ${RENDER}"
+```
+
+#### 2.2 Run Container
+```bash
+docker run --rm \
+    -v .:/workspace \
+    --group-add ${VIDEO} \
+    ${RENDER_GROUP} \
+    --device=/dev/dri \
+    --ipc=host \
+    -e http_proxy=$http_proxy \
+    -e https_proxy=$https_proxy \
+    -e no_proxy=$no_proxy \
+    -it $IMAGE_NAME bash
+```
+
+#### 2.3 Environment Settings
+Source installed torch 2.0.1+gpu is required. Please refer to this [tutorial](https://github.com/pytorch/pytorch#from-source) to check system requirements and and install dependencies.
+```bash
+git clone https://github.com/pytorch/pytorch.git
+```
+
+Please set basekit configurations as following:
+```bash
+bash l_BaseKit_p_2024.0.0.49261_offline.sh -a -s --eula accept --components intel.oneapi.lin.tbb.devel:intel.oneapi.lin.ccl.devel:intel.oneapi.lin.mkl.devel:intel.oneapi.lin.dpcpp-cpp-compiler --install-dir ${HOME}/intel/oneapi
+source ./20240921_xmainrel/env/vars.sh
+# source ${HOME}/intel/oneapi/compiler/latest/env/vars.sh
+source ${HOME}/intel/oneapi/mkl/latest/env/vars.sh
+source ${HOME}/intel/oneapi/tbb/latest/env/vars.sh
+export MKL_DPCPP_ROOT=${MKLROOT}
+export LD_LIBRARY_PATH=${MKL_DPCPP_ROOT}/lib:${MKL_DPCPP_ROOT}/lib64:${MKL_DPCPP_ROOT}/lib/intel64:${LD_LIBRARY_PATH}
+export LIBRARY_PATH=${MKL_DPCPP_ROOT}/lib:${MKL_DPCPP_ROOT}/lib64:${MKL_DPCPP_ROOT}/lib/intel64:$LIBRARY_PATH
+```
+Source installed IPEX is required. Please refer to this [tutorial](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.0.110%2Bxpu) to check system requirements and install dependencies.
+```bash
+git clone https://github.com/intel/intel-extension-for-pytorch.git
```

Comment:
Source build branch `main` branch? 
Please specify a commit until the former release is available.